.github/workflows/build-and-publish.yml

```diff
@@ -51,6 +51,8 @@ jobs:
 
       - name: Configure AWS credentials
         uses: ./.github/actions/configure-aws-credentials
+        env:
+            TF_LOG: DEBUG
```

Comment:
not sure if this is the right place or if it should be in the `configure-aws-credentials/action.yml` file instead

Reply:
Try putting it in that file instead

Reply:
Updated @coilysiren